package.json

```diff
@@ -11,7 +11,18 @@
   "configSchema": {
     "executablePath": {
       "type": "string",
-      "default": "codeclimate"
+      "default": "codeclimate",
+      "description": "The path to the `codeclimate` executable."
```

Comment:
"The path" -> "The full path" seems to help users a little bit, I've yet to find a wording for this that is clear to everyone 😞.